pkg/cephmgr/osd/partition/scheme.go

```diff
@@ -32,78 +33,96 @@ const (
 	databasePartition     = 2
 	blockPartition        = 3
 	UseRemainingSpace     = -1
-	SimpleVersion         = 1
 	schemeFilename        = "partition-scheme"
+	WalDefaultSizeMB      = int64(576)
```

Comment:
you're right, there's not a need to use a 64 bit value when the units are MB.  i will change the type to 32 bit, but keep the MB suffix because from a readability and usability standpoint i like the units to be clear in the var naming.